mono/mini/mini-amd64.c

```diff
@@ -1225,6 +1236,10 @@ mono_arch_cpu_optimizations (guint32 *exclude_mask)
 		*exclude_mask |= MONO_OPT_CMOV;
 	}
 
+#ifdef TARGET_WIN32
+	*exclude_mask |= MONO_OPT_SIMD;
```

Comment:
The current SIMD doesn't support the argument used by a LD_ADDR to be of type OP_VTARG_ADDR. This will now be used for value types > 8 or of size 3,5,6,7 as dictated by windows x64 value type ABI. Since OP_VTARG_ADDR needs to be resolved in mono_spill_global_vars and the SIMD implementation optimize away the LD_ADDR in load_simd_vreg, that will cause an error in mono_spill_global_vars since incorrect opcode will now have a reference to an argument that won't be fully resolved. I looked into just resolving the optimization but it turns out that there are several constructs that could end up with different opcodes having the same problem emitted by the SIMD optimizations. I discussed this with @vargaz and the recommendation was to disable this on win x64 for now and look into this later, if needed.